### DISCUSSION TITLE
FirebaseError: Firebase: Error (auth/argument-error).

### DISCUSSION THREAD

**Discussion by maxlibin** [gladly-team/next-firebase-auth#337] [Q&A] [answered]
**Describe the bug**
Any idea why I am getting FirebaseError: Firebase: Error (auth/argument-error). when I use `withAuthUser`
<img width="1284" alt="Screenshot 2021-11-07 at 8 09 21 PM" src="https://user-images.githubusercontent.com/1277038/140644316-dead8ee1-7614-4699-b29b-92650e15564d.png">

I am using firebase 9, and so I have installed: `next-firebase-auth@canary`, the problem occured on Mac only, is fine on Linux.

```
export default withAuthUser({
  whenAuthed: AuthAction.REDIRECT_TO_APP,
  whenUnauthedBeforeInit: AuthAction.SHOW_LOADER,
  whenUnauthedAfterInit: AuthAction.RENDER,
  LoaderComponent: Loader,
})(Login);
```

**Comment by kmjennison** [comment, marked as answer]
This could be caused by an incorrect email in Firebase credentials. See [troubleshooting](https://github.com/gladly-team/next-firebase-auth/blob/v1.x/README.md#i-get-an-authargument-error-with-message-firebase-id-token-has-invalid-signature).

If that doesn't solve it, please share any other error logs or anything else you've tried so far.
